File: scratch/test_bugs.mjs
import { chromium } from 'playwright';

(async () => {
  const browser = await chromium.launch();
  const page = await browser.newPage();
  
  page.on('console', msg => {
    if (msg.type() === 'error') {
      console.log(`[Browser Error]: ${msg.text()}`);
    }
  });

  page.on('pageerror', error => {
    console.log(`[Page Error]: ${error.message}`);
  });

  try {
    console.log("Navigating to Inventory...");
    await page.goto('http://localhost:5173/inventory', { waitUntil: 'networkidle', timeout: 5000 });
  } catch (e) {
    console.log(`Inventory navigation error: ${e.message}`);
  }

  try {
    console.log("Navigating to Reports...");
    await page.goto('http://localhost:5173/reports', { waitUntil: 'networkidle', timeout: 5000 });
  } catch (e) {
    console.log(`Reports navigation error: ${e.message}`);
  }

  try {
    console.log("Navigating to License Compliance...");
    await page.goto('http://localhost:5173/drivers/compliance', { waitUntil: 'networkidle', timeout: 5000 });
  } catch (e) {
    console.log(`License navigation error: ${e.message}`);
  }

  await browser.close();
})();

File: scratch/test_bugs.py
from playwright.sync_api import sync_playwright

def run(playwright):
    browser = playwright.chromium.launch()
    page = browser.new_page()

    # Capture console messages
    page.on("console", lambda msg: print(f"[{msg.type}] {msg.text}"))
    page.on("pageerror", lambda err: print(f"[PageError] {err}"))

    print("Navigating to Inventory...")
    try:
        page.goto("http://localhost:5173/inventory/restock", timeout=5000, wait_until="networkidle")
    except Exception as e:
        print(f"Inventory nav error: {e}")

    print("Navigating to Reports...")
    try:
        page.goto("http://localhost:5173/reports", timeout=5000, wait_until="networkidle")
    except Exception as e:
        print(f"Reports nav error: {e}")

    print("Navigating to License Compliance...")
    try:
        page.goto("http://localhost:5173/drivers/license-compliance", timeout=5000, wait_until="networkidle")
    except Exception as e:
        print(f"License nav error: {e}")

    browser.close()

with sync_playwright() as playwright:
    run(playwright)
